### DISCUSSION TITLE
Catch all route overwritten by 404.vue

### DISCUSSION THREAD

**Discussion by vweltje** [nuxt/nuxt#16421] [Questions]
I am trying to implement a 404 page into my project. But as soon as I add 'pages/404.vue' my catch all route is never been used anymore and the 404 page will be shown instead. 

I followed the steps listed on the documentation page: https://v3.nuxtjs.org/guide/directory-structure/pages/#catch-all-route
Using nuxt `3.0.0-rc.12`

My folder structure:
```
pages
  [...slug].vue
  404.vue
```
    
What am I missing?

**Comment by TheAlexLichter** [comment]
Related: https://github.com/nuxt/framework/issues/6895#event-7771216619
